src/main/java/racingcar/Winners.java

```diff
@@ -0,0 +1,30 @@
+package racingcar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+    private static final int MIN_POSITION = 0;
+
+    private final List<Car> winners;
+
+    public Winners(List<Car> cars) {
+        int maxPosition = getMaxPosition(cars);
+        winners = cars.stream()
+                .filter(car -> car.position() == maxPosition)
```

Comment:
```suggestion
                .filter(car -> car.sameOf(maxPosition))
```
값을 꺼내기보단 상태를 가진객체에게 직접 비교하도록,
이런 메시지를 보내보는것은 어떤가요?

Reply:
좋습니다~!